W-16484122-github-non-enterprise-account-types-fa (#239)
mulesoft/docs-design-center: modules/ROOT/pages/design-ghs-install-authorize.adoc

* You cannot install and use GitHub Synchronization if your GitHub account uses an IP allow list. 

* In the email settings for the GitHub account you want to use with GitHub Synchronization, if you select *Keep my email addresses private*, ensure you do not select *Block command line pushes that expose my email*. The block setting prevents you from pushing commits in your synchronized project.

> * The GitHub integration supports only non-enterprise accounts. For more information, refer to https://docs.github.com/en/enterprise-cloud@latest/get-started/learning-about-github/types-of-github-accounts[Types of Github Accounts].

Reviewer: ```suggestion
* The GitHub integration supports only non-enterprise accounts. See https://docs.github.com/en/enterprise-cloud@latest/get-started/learning-about-github/types-of-github-accounts[Types of Github Accounts].
```
